### PR TITLE
Add role membership workflow handlers

### PR DESCRIPTION
**Pull request by tjerman**

### CODE REVIEW

File: system/automation/roles_handler.yaml [hunk 1]
```diff
@@ -76,6 +86,50 @@ functions:
     results:
       role: *rvRole
 
+  lookupMembers:
```
Comment (darh) [inline system/automation/roles_handler.yaml]: you either search for members (plural) or lookup for a member (singular)
so this is probably searchMembers, correct?

File: tests/workflows/testdata/S1001_role_members/data_model.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+users:
```
Comment (darh) [inline tests/workflows/testdata/S1001_role_members/data_model.yaml]: the idea for test numeration was to just do +1, nothing more than that.

**Review by darh — CHANGES_REQUESTED** [review]

File: system/automation/roles_handler.yaml (hunk 1 above)
Comment (tjerman) [reply system/automation/roles_handler.yaml]: True

File: tests/workflows/testdata/S1001_role_members/data_model.yaml (hunk 2 above)
Comment (tjerman) [reply tests/workflows/testdata/S1001_role_members/data_model.yaml]: Noted

**Review by darh — APPROVED** [review]
:shipit:

**Comment by tjerman** [comment]
Merged; 3 and 9

**Pull request closed without merging**